contracts/Recruitment.sol

```diff
@@ -73,6 +77,27 @@ contract Recruitment {
     return referredEmails[msg.sender];
   }
   
+
+
+  struct ReferralScore {
+        uint256 score;  //Score given by the hiring company to the candidate 
+        address senderAddress; // Wallet address of the hiring company
+  }
+  event ReferralScoreSubmitted(address senderAddress, address referrerWallet, uint256 score);
+  mapping(address => ReferralScore[]) public referralScores;
+
+  function submitReferralScore(uint256 score,address referrerWallet) public returns (bytes32) {
+    require(msg.sender == owner, 'Only owner can submit referral score!');
```

Comment:
I think this is not going to work. Isn't it that if we do this only the smart contract owner can give scores?